queries/comment/highlights.scm

```diff
@@ -8,16 +8,16 @@
 (tag (name) @text.note (user)? @constant)
 
 ((tag ((name) @text.warning))
- (#match? @text.warning "^(TODO|HACK|WARNING)$"))
+ (#any-of? @text.warning "TODO" "HACK" "WARNING)$"))
```

Comment:
```diff
-  (#any-of? @text.warning "TODO" "HACK" "WARNING)$"))
+  (#any-of? @text.warning "TODO" "HACK" "WARNING"))
```

Reply:
Good catch! if you want to make a PR with that fix, go for it!